controller/lib/core/sensors.h

```diff
@@ -111,4 +119,17 @@ void set_sensor_avg_samples(int numAvgSamples);
  */
 int get_sensor_avg_samples();
 
+/*
+ * @brief Method implements Bernoulli's equation assuming the Venturi Effect.
+ * Solves for the volumetric flow rate since A1/A2, rho, and differential
+ * pressure are known. Q = sqrt(2/rho) * (A1*A2) * 1/sqrt(A1^2-A2^2) *
+ * sqrt(p1-p2); based on (p1 - p2) = (rho/2) * (v2^2 - v1^2); where A1 > A2
+ * @param diffPressureInKiloPascals the differential pressure from a sensor in
+ * [kPa]
+ * @return the volumetric flow in [meters^3/s]. Can be negative, indicating
+ * direction of flow, depending on how the differential sensor is attached to
+ * the venturi.
+ */
+float convert_diff_pressure_to_volumetric_flow(float diffPressureInKiloPascals);
```

Comment:
I wonder if this function belongs in the "sensors" file at all?  It seems to be a pure function, unlike the other sensor code.

Reply:
Yeah I was thinking the same. Though I felt since it was only going to be called in reference to getting pressure data from the sensors...it should live with the code that uses it. I don't see anyone else needing to convert differential pressure to Q that doesn't involve taking a reading from the sensors.